ARGO/include/GameScreen.h

```diff
@@ -36,15 +36,14 @@ class GameScreen
 	void updateProjectiles(float t_deltaTime);
 	void updateLevelManager();
 	void setControllerButtonMap(ButtonCommandMap t_controllerMaps[Utilities::NUMBER_OF_CONTROLLER_MAPS][Utilities::S_MAX_PLAYERS]);
-	void removeDeadEnemies();
 
 	EventManager& m_eventManager;
 
 	static const int MAX_ENTITIES = 10000;
 
 	Controller m_controllers[Utilities::S_MAX_PLAYERS];
 	Entity m_players[Utilities::S_MAX_PLAYERS];
-	std::vector<Entity> m_entities;
+	//std::vector<Entity> m_entities;
```

Comment:
Do you need this?

Reply:
I knew I'd miss one of the things I commented out